pkg/vm/builtins/builtin_runner.go

```diff
@@ -25,6 +25,7 @@ const (
 	AddModeType
 	MulModType
 	GasBuiltinType
+	SystemType
```

Comment:
What does this one refer to? I see you only use it in emulatedBuiltins?

Reply:
ah yes, this is something I thought might be useful in the future, but it turned out not

Reply:
Can it be removed then?

Reply:
removed